webapp/src/pages/ChatRoom/MessageInput.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Input, InputVariant } from '../../shared/components';
+import socket from '../../shared/socket';
+import './MessageInput.css';
+
+const NewMessage = () => {
+  const [value, setValue] = useState('');
+  const submitForm = (e: React.FormEvent) => {
+    e.preventDefault();
+    socket.emit('message', value);
+    setValue('');
+  };
+
+  return (
+    <form className="message-input-form" onSubmit={submitForm}>
+      <Input
+        variant={InputVariant.DARK}
```

Comment:
What do we think about Making this input component have the text wrap and have the height of the input field increase at each new line appear? I mean something like on desktop facebook.
Something like [this](https://stackoverflow.com/questions/69562816/wrapping-and-height-increasing-input-react-css) suggests

Reply:
I like it, with maximun of "x" lines, and then scroll. We can try it

Reply:
I like it too :+1:

Does this mean we should also add a button to submit the message since pressing <kbd>Enter</kbd> will add a new line?

Reply:
I added this change. Please take a look, and if you have any suggestions, please let me know.

Reply:
Both in Firefox and in Chrome, I'm seeing that the stuff I paste in the input field is being pasted with format (??)
![Screenshot from 2022-10-07 12-24-25](https://user-images.githubusercontent.com/2774420/194521016-be2fba93-9b66-466b-89f6-61e6d11113c5.png)
![Screenshot from 2022-10-07 12-24-10](https://user-images.githubusercontent.com/2774420/194521022-0f1a6aa3-51f5-40a4-b7e2-8821c4064c1b.png)
![Screenshot from 2022-10-07 12-23-42](https://user-images.githubusercontent.com/2774420/194521025-5d91116e-5495-4bca-9d7b-3aaae90bae86.png)
???

Reply:
Good catch! Thanks!

I'm looking into how we could fix this, and so far, I didn't find any viable solution.

Reply:
I think it's fixed now. I've added the trick from this article -> https://css-tricks.com/the-cleanest-trick-for-autogrowing-textareas/